extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -155,5 +162,37 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		onAdLoadCallback(element.getId(), gptEvent, iframe);
 	};
 
+	GoogleTag.prototype.destroySlots = function (slotsNames) {
+		var slotsToDestroy = [], success, allSlots = window.googletag.getSlots();
+
+		slotsNames.forEach(function(slotName) {
+			allSlots.forEach(function(slot) {
+				if (adUnitsSlotsMap[slot.getAdUnitPath()] === slotName) {
+					slotsToDestroy.push(slot);
+				}
+			});
+		});
+
+		if (slotsToDestroy.length) {
+			log(['destroySlots', slotsNames], 'debug', logGroup);
+			success = window.googletag.destroySlots(slotsToDestroy);
+
+			if (!success) {
+				log(['destroySlots', slotsNames, 'failed'], 'error', logGroup);
+			}
+		}
+
+		slotsToDestroy.forEach(function(slot) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)